docs/manuals/inrichting-zaakafhandelcomponent/inrichting-zaakafhandelcomponent.md

```diff
@@ -199,6 +199,13 @@ Voor nu worden de volgende onderdelen gecheckt:
   - Informatieobjecttype | er wordt voor de werking van de ZAC gecheckt of het zaaktype aan de vereiste informatieobjecttypen is gekoppeld. Dit is momenteel ‘e-mail’ dat gebruikt wordt voor het als document toevoegen van vanuit de ZAC verzonden e-mails.
   - Besluittype | er wordt gecheckt of aan het zaaktype een besluittype is gekoppeld. Dit gebeurt alleen als aan het zaaktype een resultaattype is toegevoegd dat als afleidingswijze de begin- of vervaldatum van een besluit heeft.
 
+### Roltypen
+ZAC zoekt naar een Roltype met behulp van een van deze velden:
+- Omschrijving generiek | een set vooraf gedefinieerde waarden
+- Omschrijving | beschrijving van de roltype
+
+ZAC zoekt eerst in `Omschrijving` en daarna in `Omschrijving generiek`.
```

Comment:
```suggestion
ZAC zoekt eerst in `Omschrijving` en daarna in `Omschrijving generiek`. Als voor een roltype zowel het `omschrijving` als het `omschrijvingGeneriek` veld gevuld is dan wordt het `omschrijving` veld gebruikt.
```

Reply:
Good one. Thanks!

We should decide if we go with UI description or API fields here though, before I change it

Reply:
let's do human readable texts indeed